code/modules/mob/living/living.dm

```diff
@@ -1723,6 +1756,12 @@
 	. = usable_hands
 	usable_hands = new_value
 
+	if(new_value > .) // Gained hand usage.
```

Comment:
This is really quite ugly, I think you'd be better off returning usable_hands and doing the setting later on.

Reply:
This is done so that redefinitions of the proc can call parent first and know if a change happened, and what kind of change.